contributing.md

```diff
@@ -512,6 +512,97 @@ coverage report. You'll need to have [lcov](http://ltp.sourceforge.net/coverage/
 
         firefox coverage/index.html
 
+### Sanitizers
+
+Sanitizers capture very detailed information during runtime about code quality issues and print them to stderr.
+
+#### Install dependencies
+
+Install and enable Colcon mixin package:
+
+```bash
+sudo apt-get install python3-colcon-mixin
```

Comment:
Ignition developers may not have ROS packages enabled. How about installing from PIP, like we do on Ignition's [source install instructions](https://ignitionrobotics.org/docs/citadel/install_ubuntu_src#vcstool-and-colcon-from-pip)?

 ```suggestion
pip3 install colcon-mixin
```